README.md

```diff
@@ -1,2 +1,3 @@
 # hello-world
 This is my first step to learn how things work in github
+This is the first commit/change that is being made on one of my branch named readme-updates
```

Comment:
This is a test comment under the line where changes were made and then committed.